src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -334,29 +354,32 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     }
 
     /**
-     * @param array<string, string> $uses
+     * @param _Uses $uses
      */
     private function refreshUsesCache(array $uses): void
     {
-        if ($this->cacheUsesLast === $uses) {
-            return;
-        }
+        foreach ($uses as $kind => $kindUses) {
```

Comment:
the foreach must be put right before the other foreach

Reply:
Can you explain a bit more?

Reply:
It is evident, the lines after the first `foreach` are unrelated to the loop, ie. must be placed before the loop.

Reply:
@Wirone I think the merged change is still wrong - ll413-414 must go before 411